unit-test/joint_test.cpp

```diff
@@ -27,7 +27,7 @@
 DOCTEST_TEST_CASE("joint reactions")
 {
 	b2Vec2 gravity(0, -10.0f);
-	b2World world = b2World(gravity);
+	b2World world{gravity};
```

Comment:
This used to call a copy ctor. Was harmless, but no longer compiled after this change (unless you enabled C++17 or newer).